.github/workflows/tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Specs
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Specs
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node: [10.x, 12.x, 14.x]
```

Comment:
Instead of `.x` here should be first LTS versions. And here should be commented `15` which should be uncommented after fixing an NPM bug or adding a fallback on the `core-js` side.

Reply:
Example:
https://github.com/pustovalov/core-js/actions/runs/332170920
https://github.com/pustovalov/core-js/actions/runs/332170917